src/ray/object_manager/object_manager.cc

```diff
@@ -931,8 +931,7 @@ void ObjectManager::RecordMetrics() const {
   stats::ObjectStoreAvailableMemory().Record(config_.object_store_memory - used_memory_);
   stats::ObjectStoreUsedMemory().Record(used_memory_);
   stats::ObjectStoreFallbackMemory().Record(
-      // TODO(scv119): this is not thread safe.
-      plasma::PlasmaAllocator::GetInstance().FallbackAllocated());
+      plasma::plasma_store_runner->GetFallbackAllocated());
```

Comment:
If `GetFallbackAllocated ` is blocking, I feel like this code can cause some issues when the event queue is saturated.

Wonder if we should just use callback here lol (plasma::plasma_store_runner->GetFallbackAllocated([](size_t allocated) {   stats::ObjectStoreFallbackMemory().Record(allocated) }))